### DISCUSSION TITLE
Welcome to Reddit-Username-Sniper Discussions!

### DISCUSSION THREAD

**Discussion by xlyoaz** [xlyoaz/Reddit-Username-Sniper#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by reddit-mai** [comment]
Hello!, i'm mai, i am 21 Currently, i am just here because i was seriously looking to snipe a rare reddit username, i wanted to talk to you on discord about it but idk if you have it or not but yea..
